index.bs

```diff
@@ -187,7 +188,27 @@ Presentation API Requirements {#requirements-presentation-api}
 Remote Playback API Requirements {#requirements-remote-playback}
 ----------------------------------------------------------------
 
-Issue(3): Requirements for Remote Playback API.
+1.  A [=controlling user agent=] must be able to find out whether there is at
+    least one compatible [=remote playback device=] available for a given
+    {{HTMLMediaElement}}, both instantaneously and continuously.
+
+2.  A controlling user agent must be able to start sending the content of an
```

Comment:
"sending the content" is a bit vague. Is this defined somewhere else?

Reply:
I rephrased this to use the term _initiate remote playback_ from the Remote Playback spec which seems to be the closest in meaning.